README.md

```diff
@@ -56,12 +56,13 @@ Have a bug or a feature request? [Please open a new issue](https://github.com/Ly
 * [kamil4](https://github.com/kamil4)
 * [ildyria](https://github.com/ildyria)
 * [nagmat84](https://github.com/nagmat84)
+* [qwerty287](https://github.com/qwerty287)
```

Comment:
The ordering of this list feels wrong. Perhaps we should copy from/link to https://lycheeorg.github.io/support.html ?